[Doc] Replace tensor by field for files beginning with o to s (#1578)
taichi-dev/taichi: docs/snode.rst

.. function:: snode.place(x, ...)

    :parameter snode: (SNode) where to place

>     :parameter x: (field) field(s) to be placed

Reviewer: r

Reviewer: ```suggestion
    :parameter a: (ti.field) field(s) to be placed
```

        assert x.snode() == y.snode()

.. function:: field.shape

>     :parameter field: (Field)

Reviewer: r

Reviewer: ```suggestion
    :parameter a: (ti.field)
```

        x.shape # returns (3, 5, 4)

.. function:: field.snode()

>     :parameter field: (Field)

Reviewer: r

Reviewer: ```suggestion
    :parameter a: (ti.field)
```
